frontend/src/pages/interviewschedule.js

```diff
@@ -121,14 +178,46 @@ class InterviewSchedule extends Component {
           <form onSubmit={this.uploadSchedule}>
             <>
               <h2>Upload a New Schedule</h2>
+              <h4>Candidates</h4>
+              <input
+                ref={ref => {
+                  this.candidateInput = ref
+                }}
+                onChange={_ => this.setState({ candidateFileSelected: true })}
```

Comment:
out of curiosity, what's the diff between `() =>` and `_ => `?

Reply:
`()` implies the signature takes in no params, while `_` implies that it takes in some param (ex. `(a) =>`) but to discard that param rather than store it in some variable since it's not used.